src/include/splash/SerialDataCollector.hpp

```diff
@@ -61,10 +61,11 @@ namespace splash
         /**
          * Constructs a filename from a base filename and the process' mpi position
          * such as baseFilename+mpiPos+.h5
+         * Does nothing if the filename already contains a ".h5"
```

Comment:
yes, just add to the comment that single-file is only allowed with Prod(mpiSize) == 1 (see corruption argument above)